pkg/resource/template/template_funcs.go

```diff
@@ -33,6 +35,7 @@ func newFuncMap() map[string]interface{} {
 	m["fileExists"] = isFileExist
 	m["base64Encode"] = Base64Encode
 	m["base64Decode"] = Base64Decode
+	m["hash"] = hashToIPv4
```

Comment:
Nit: please can you use the same name for the template function as you use for the golang function.  I.e. `hash["hashToIPv4"] = hashToIPv4` (you'll need to change the template too).

Reply:
ack